raiden-dapp/vue.config.js

```diff
@@ -89,12 +93,14 @@ module.exports = {
         }),
       );
     }
-  },
-  pwa: {
-    workboxPluginMode: 'InjectManifest',
-    workboxOptions: {
-      swSrc: './src/sw.js',
-      swDest: 'service-worker.js',
-    },
+
+    if (process.env.NODE_ENV === 'production') {
```

Comment:
Mh, should we maybe enable this on staging as well for testing purposes?

Reply:
Well, you can do this manually if you really need it. Though I don't recommend it. 
For staging this does not work as this would prevent updates. Staging version are only between to package versions/releases. This means once you loaded the first time a staging version after a new release, this will be sticky until the next release has passed. Means you won't see any other staging version in-between.
For testing I think we can't. Unit tests don't have this environment available. And the end-to-end tests in Cypress lack in control of the environment. Means we don't get a clean setup. I somewhere documented. Though we can (and probably should) try to get it working in the CI, hoping that the cache we have (yes includes Cypress) does not include such stuff. @taleldayekh how did you solved the localStorage issue back then? We once had the issue that Cypress does not clear the data between test runs.

Reply:
It was solved like this: https://github.com/raiden-network/light-client/blob/eb712da309d6c2ed22f93cf657121d53b08bbba2/raiden-dapp/tests/e2e/specs/scenario.spec.ts#L54

Reply:
So we eventually could do this for the Cache API and IndexedDB as well.  Though this will not test the service worker its features, but just makes sure nothing is broken or unexpected updates are proposed. The Cypress environment is not able to test the update procedure. At least not with a feasible amount of work (I discussed this with Christian).
I would love to have a talk with us 3 to evaluate what we could do best for making sure nothing gets broken over time. Apparently I forgot many things. But the right questions should help.

Reply:
See [this comment](https://github.com/raiden-network/light-client/pull/2455#issuecomment-765267101).